.github/workflows/publish-action.yml

```diff
@@ -6,61 +6,13 @@ on:
       - "main"
 
 jobs:
-  # haven't found a way to reuse the locally defined action.yml at the root of the repo
   release:
     name: Release
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@master
-      - name: GitHub Tag Bump
-        uses: anothrNick/github-tag-action@1.36.0
-        id: tag
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-          DEFAULT_BUMP: ${{ inputs.default-bump }}
-          WITH_V: true
-          DRY_RUN: true
-      - name: List Release
-        run: echo "This will create the release '${{ steps.tag.outputs.new_tag }}'"
-      - name: Create Release
-        uses: ncipollo/release-action@v1
+        uses: actions/checkout@v3
+      - name: GitHub semver release
+        uses: vivantehealth/github-semver-release-action@v0
```

Comment:
it uses itself? no chicken/egg problems here, now that v0 exists? what if v0 breaks?

Reply:
that makes sense. thinking about it a bit more, i'm also wondering how we'd change the behavior if we need to modify the functionality. unless i'm missing something, we're losing visibility into how it works, but also losing the ability to easily make changes in the future

Reply:
Why is that?
When I make changes here, I fork the repo and test them in them in the fork.  
I don't understand how using a reference removes the ability to make changes in the future given that changing this reference when developing testing is very easy.

Reply:
maybe i'm misunderstanding the effect of this change, or your workflow. what would your workflow be to change the github token, for example? or add some logic to do a semver release but without the `v` prefix?